### ISSUE TITLE
When a plasmid stock is made from a glycerol stock, it is "unverified"

### ISSUE THREAD

**Issue by dyounger** [klavinslab/aquarium-bugs#174]
It would be great if a new plasmid stock made from a validated glycerol stock would make a verified plasmid stock instead of an unverified one (this will hopefully be a non-issue with later updates, but it has come up a few times and I think it should be an easy fix.)

Thanks!

**Comment by gasnew** [comment]
Yep, this will be addressed very soon! This fix dovetails with fixes for #152, #142, #97, and #96 that will come very shortly after we get modules.
